packages/mask/src/plugins/Savings/SNSAdaptor/protocols/index.ts

```diff
@@ -0,0 +1,20 @@
+import type { SavingsProtocol } from '../../types'
+import { lidoDeposit, lidoWithdraw } from './LDOProtocol'
+
+export const SavingsProtocols: SavingsProtocol[] = [
+    {
+        id: 0,
+        name: 'Lido',
+        image: 'lido',
+        base: 'ETH',
+        pair: 'stETH',
+        apr: '5.3',
```

Comment:
Why is the apr a fixed value? Should it fetch from contract or external APIs?

Reply:
The APR fetched is from `getLidoAPR()` in `SavingsDialog.tsx`.
However, there is still a CORS issue with the CORS anywhere plugin you suggested.

I'm pretty sure that Lido blocks any request to it's website that isn't it's origin.
Do you have any suggestions of where to fetch the actual APR value?

I can't seem to find a reliable source for this that is non trivial.